translator/cmdutil/userutil_linux_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+// +build linux
```

Comment:
Why will this test not run on mac? Can you just post the error message you get out here. I'm assuming windows file permission system is different. I do not know much about windows.

Reply:
To run the test on mac, you would have to include darwin in the build tag. https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
There's no error message. It just won't get picked up.